edit/addproj.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 01/10/2018
+ * Time: 10:21
+ */
+
+?>
+
+<html>
+<head>
+    <title>Edit Badges</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <form>
+            <ul>
+                <li>
```

Comment:
do you need the bullet point list? shouldnt do

Reply:
fixed in 90c0c0f